docs/snippets/rendering_dump_variable.md

```diff
@@ -0,0 +1,8 @@
+!!! tip
+
+    For development purposes, you can view all available variables, or a single variable, and their values, using the `dump()` Twig function:
```

Comment:
```suggestion
    For development purposes, you can list all available variables, or a single variable, and their values, by using the `dump()` Twig function:
```

view -> display?
You can't view them until you get them listed on the screen with the dump() function.